lib/perl/Genome/Model/Tools/Manta/Base.pm

```diff
@@ -0,0 +1,246 @@
+package Genome::Model::Tools::Manta::Base;
+
+use strict;
+use warnings;
+
+use Genome;
+use Carp qw(confess);
+
+class Genome::Model::Tools::Manta::Base {
+    is => 'Command',
+    is_abstract => 1,
+    attributes_have => [
+        tool_arg_name => {
+            is => 'String',
+            is_optional => 1,
+        },
+        tool_input_file => {
+            is => 'Boolean',
+            is_optional => 1,
+        },
+    ],
+    has_param => [
+        version => {
+            is => 'Version',
+            doc => 'The version of Manta to use.',
+            valid_values => ['0.29.6'],
+        },
+    ],
+    has_optional_output => [
+        output_files => {
+            is_many => 1,
+        },
+        input_files => {
+            is_many => 1,
+        },
+    ],
+};
+
+sub help_detail {
+    "Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads."
+}
+
+my %TOOL_VERSIONS = (
+    '0.29.6'     => '/gscmnt/gc13001/info/model_data/jwalker_scratch/src/manta-0.29.6.centos5_x86_64/bin',
```

Comment:
@mcallaway the config env variable would have to be specific to each version, ex. `GENOME_TOOL_EXEC_PATH_MANTA_0.29.6` in order to support multiple versions of a tool.  Encoding the software version in the env doesn't feel right.  Using a container rather than packaging seems like a better option, but see comment below.

@sleongmgi please provide an example of how to setup a GMT to run in a docker container inside of DV2 as part of a genome model build.  Using a docker container for the first time seems outside the scope of this PR.  Feel free to submit a pull request to my branch if it's something simple that I'm overlooking.  I'm all for utilizing docker if it's straightforward, but developing the GMS to support it is again outside the scope of this issue, ie. replacing breakdander for SVs.